packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@readme/eslint-plugin",
+  "version": "1.0.0",
+  "description": "ESLint plugin providing custom rules for ReadMe's coding standards",
+  "main": "index.js",
+  "author": "Jon Ursenbach <jon@ursenba.ch>",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/readmeio/standards.git",
+    "directory": "packages/eslint-plugin"
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "prettier": "prettier --list-different --write \"./**/**.js\"",
+    "lint": "eslint .",
```

Comment:
up to you here but if you wanted to bring in [`eslint-define-config`](https://github.com/Shinigami92/eslint-define-config) as a dev-dep to do a lightweight type check (see https://github.com/readmeio/standards/pull/753 for how I did this) that might be helpful!

Reply:
Running into a bunch of typing issues on our rule code with this being present that aren't worth fixing now. When ESLint 9.0 hits later this year I'd rather take some time to move all of our configs and this new plugin over to TS.

<img width="1631" alt="Screen Shot 2023-10-31 at 1 50 51 PM" src="https://github.com/readmeio/standards/assets/33762/04a2ac62-40d7-4049-9772-c6c8aad4d2e7">